src/GraphQL/Resolver/StartUploadResolver.php

```
@@ -0,0 +1,212 @@
+                $result->userErrors,
+            ),
+        ];
+    }
+
+    private function accountId(mixed $context): string
+    {
+        if (is_array($context) && isset($context['accountId']) && is_string($context['accountId'])) {
+            return $context['accountId'];
+        }
+
+        throw MissingAccountContextException::missing();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\GraphQL\Exception\MissingAccountContextException' in method 'accountId'. (undefined)

(StaticAccess)